modules/ibm-z-secure-execution.adoc

```diff
@@ -112,13 +112,19 @@ To achieve the desired security, Ignition logging and local login are disabled b
 ----
 gpg --recipient-file /path/to/ignition.gpg.pub --yes --output /path/to/config.ign.gpg --verbose --armor --encrypt /path/to/config.ign
 ----
+
+. Follow the fast-track installation procedure to install nodes using the {ibm-name} Secure Execution QCOW image.
 +
 [NOTE]
 ====
-Before starting the VM, replace `serial=ignition` with `serial=ignition_crypted` when mounting the Ignition file.
+Before starting the VM, replace `serial=ignition` with `serial=ignition_crypted` and add the `launchSecurity` parameter when mounting the Ignition file.
```

Comment:
`launchSecurity` affects all IO deives and must be enabled on each VM boot. I guess it'd be better to drop `when mounting the Ignition file.`